managing-os/pentesting.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: doc
+title: Penetration Testing
+permalink: /doc/pentesting/
+---
+
+**Legal notice:**
+
+The usage of penetration testing tools outside your own laboratory environment requires the permission of the organization you attack. Penetration testing without such a permission can have legal consequences.
+
+To avoid such legal conflicts please refer to the [Code of Ethics](https://www.eccouncil.org/Support/code-of-ethics).
+
+Penetration Testing
+===================
+
+"A penetration test, informally pen test, is an attack on a computer system that looks for security weaknesses, potentially gaining access to the computer's features and data." (source [Penetration test](https://en.wikipedia.org/wiki/Penetration_test)).
+
+Penetration Testing Distributions:
+----------------------------------
+
+The following install instructions explain how to setup a penetration testing distribution within Qubes OS.  
+
+- [BlackArch](/doc/blackarch/)
+- [Kali](/doc/kali/)
+- [PenTester Framework (PTF)](/doc/ptf/)
```

Comment:
Since you're linking to all of the Pentesting pages here, it might be better not to link to them from the main `doc.md` ToC. In other words, follow the lead of the Whonix pages, not the Template pages (which are different, because they're the main, official, Qubes-supported templates). @marmarek, thoughts on this?